test/Vagrantfile

```diff
@@ -2,15 +2,20 @@
 # vi: set ft=ruby
 Vagrant.require_version ">= 2.2.0"
 
+# The source of truth for vagrant box versions.
+# Sets SERVER_BOX, SERVER_VERSION, NETNEXT_SERVER_BOXET and NEXT_SERVER_VERSION
+# Accepts overrides from env variables
+require_relative '../vagrant_box_defaults.rb'
+$SERVER_BOX = (ENV['SERVER_BOX'] || $SERVER_BOX)
+$SERVER_VERSION= (ENV['SERVER_VERSION'] || $SERVER_VERSION)
+$NETNEXT_SERVER_BOX= "cilium/ubuntu-next"
+$NETNEXT_SERVER_VERSION= "23"
```

Comment:
Shouldn't this be

```
$NETNEXT_SERVER_BOX = (ENV['NETNEXT_SERVER_BOX'] || $NETNEXT_SERVER_BOX)
$NETNEXT_SERVER_VERSION = (ENV['NETNEXT_SERVER_VERSION'] || $NETNEXT_SERVER_VERSION )
```

To be similar to how it is done for `SERVER_BOX` and `SERVER_VERSION`?

Reply:
I had gone down that path, then I realised that the NETNEXT_ variables were only used in one place. I then undid too far. Anyway, we may as well expose them as env vars, so it is as you suggest.